packages/webamp/js/reducers/tracks.ts

```diff
@@ -7,23 +7,29 @@ export interface TracksState {
 }
 
 function massageKhz(khz: number) {
-  let finalKhz: String = ' 0';
+  let finalKhz: String = " 0";
```

Comment:
Right now this doesn't make sense to include, since on playback with bigger files Webamp isn't displaying anything in the kbps and khz sections:
<img width="275" height="464" alt="image" src="https://github.com/user-attachments/assets/b907dbf9-b2ea-4408-8769-692cdf59c47e" />

Reply:
I think I found the reason why it remains empty on start:
https://github.com/captbaritone/webamp/blob/fbe3a0090f1c3f638b8d625363840a7134d85b46/packages/webamp/js/selectors.ts#L686-L698

I will have to experiment some to see if I can get this to work without breaking anything else